Using options:
fn1676_0198: error: An error occurred during data flow analysis.
    The given key 'l1676_0186' was not present in the dictionary.
      at System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) [0x0001e] in <135e7c260ae04ab1a5538976c2d8cf31>:0 
  at Reko.Core.Lib.DominatorGraph`1[T].BuildDominanceFrontiers (Reko.Core.Lib.DirectedGraph`1[T] graph, System.Collections.Generic.Dictionary`2[TKey,TValue] idoms) [0x0007c] in <b2493f84bc1c4ae0a3abd1f5c5378f54>:0 
  at Reko.Core.Lib.DominatorGraph`1[T]..ctor (Reko.Core.Lib.DirectedGraph`1[T] graph, Reko.Core.Block entryNode) [0x00034] in <b2493f84bc1c4ae0a3abd1f5c5378f54>:0 
  at Reko.Analysis.DataFlowAnalysis.BuildSsaTransform (Reko.Core.Procedure proc) [0x00035] in <6a264ef1b92c4115b517d6a392a986a9>:0 
  at Reko.Analysis.DataFlowAnalysis.BuildExpressionTrees () [0x00060] in <6a264ef1b92c4115b517d6a392a986a9>:0 
